Externals/hidapi/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+project(hidapi)
+
+set(SRCS)
```

Comment:
Would you not be able to use [target_sources](https://cmake.org/cmake/help/v3.8/command/target_sources.html) for adding the source files in this case instead of using a variable?